Token.js

```diff
@@ -643,9 +713,9 @@ class Token {
 			if (!this.options.custom_conditions.includes("Inspiration")){
 				this.options.custom_conditions.push("Inspiration")
 			}
-		}
-		else{
-			this.options.custom_conditions.pop("Inspiration")
+		} else{
+			array_remove_index_by_value(this.options.custom_conditions, "Inspiration");
+			array_remove_index_by_value(this.options.custom_conditions, "Inspiration");
```

Comment:
assuming this is fine as the second call won't find it and will just return? not looked into what `array_remove_index_by_value` does, but assuming it's a find by index and a splice?